ci/terraform/oidc/authdev1.tfvars

```diff
@@ -15,6 +15,7 @@ account_intervention_service_action_enabled = true
 account_intervention_service_abort_on_error = true
 send_storage_token_to_ipv_enabled           = true
 call_ticf_cri                               = true
+support_reauth_signout_enabled              = true
```

Comment:
Do we need to add this also to authdev2.tfvars if we want to keep our 2 dev envs synchronise?

Reply:
Yes. By default the value for this feature switch is false.